src/specklepy/objects/tests/mesh_test.py

```diff
@@ -0,0 +1,96 @@
+from devtools import debug
+from specklepy.api.operations import deserialize, serialize
```

Comment:
applicable to all test files: we can import "operations" module from core.api instead, to avoid unnecessary user-facing calls